cmd/main.go

```diff
@@ -169,6 +170,14 @@ func main() {
 	var qosMaxBuilds int
 	var qosDefaultValue int
 
+	var lffTaskQoSEnabled bool
+	var qosMaxTasks int
+	var qosMaxNamespaceTasks int
+	var qosTasksDefaultValue int
```

Comment:
Looks like this isn't used for anything? It gets a value from the `flag.IntVar` but it's never consumed.

Reply:
That was supposed to select only `qosTasksDefaultValue` as not being used.

Reply:
Ah yeah, that was going to be used for priority sorting for tasks, but was abandoned. Removed now.